### DISCUSSION TITLE
How are "most common" words selected?

### DISCUSSION THREAD

**Discussion by wch** [monkeytypegame/monkeytype#5095] [Questions] [answered]
According to the information page, Monkeytype "uses the most common 200 words in the English language" by default. But when I do a typing test, I see the word "govern", and every time I see it, I think, "how is this one of the 200 most common words in English?"

I found a word frequency list [here](https://en.wiktionary.org/wiki/Wiktionary:Frequency_lists/PG/2006/04/1-10000), and "govern" comes in at position 4079.

So all of this has me wondering: how were the 200 "most common" words in Monkeytype selected?

**Comment by Miodec** [comment, marked as answer]
To be fully honest - i just googled for a list of most common words and picked the first one. This was back towards the end of 2019 / early 2020 when I never intended for the website to be so widely used, so I didn't do much research into "correct" frequencies.

Issue with updating the word list now is that it would potentially invalidate the leaderboard completely due to a changed difficulty, but it is something i think about now and then.

**Comment by wch** [comment]
Thanks, that makes sense. As for invalidating the leaderboard... Maybe you can have leaderboard archives? For example, if you have a policy of making big changes, say, yearly, then you could have a pre-2024 leaderboard, a 2024 leaderboard, a 2025 leaderboard, etc. Every time you make big changes, you could archive the previous leaderboard.

The people who are competitive might also like having the renewed opportunity to get on the leaderboard each time you reset it. That might even argue for frequent resets -- but I don't know for sure, since I'm not one of those people.
